Update UI for ONNX models (#2)
viam-labs/onnx-cpu: README.md

Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com/).
Click on the **Services** subtab and click **Create service**.
Select the `mlmodel` type, then select the `mlmodel:onnx-cpu` model.
Click **Add module**, then enter a name for your ML model and click **Create**.

On the new component panel, deploy a model on to your machine by selecting an onnx model from the **Model** dropdown or by specifying a path to an existing model on your machine.

If you prefer to configure your service using JSON, use the following attributes **attributes**:

```json
{
  "model_path": "/path/to/onnx_file/detector_googlenet.onnx",
  "label_path": "/path/to/labels.txt",

>   "num_threads": 1

Reviewer: num_threads is not actually a config attribute for this model  - onnx models can only run on 1 thread

Author: Should it be removed from the app UI then?

Author: Because it does show up...

> }

Author: ```suggestion
  "package_reference": "viam-soleng/face-detector-onnx"
}
```

Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com/).
Click on the **Services** subtab and click **Create service**.
Select the `mlmodel` type, then select the `mlmodel:onnx-cpu` model.
Click **Add module**, then enter a name for your ML model and click **Create**.

On the new component panel, deploy a model on to your machine by selecting an onnx model from the **Model** dropdown or by specifying a path to an existing model on your machine.

If you prefer to configure your service using JSON, use the following attributes **attributes**:

```json
{
  "model_path": "/path/to/onnx_file/detector_googlenet.onnx",
  "label_path": "/path/to/labels.txt",

>   "num_threads": 1

Author: ```suggestion
  "num_threads": 1,
```

| ------- | ------ | ------------ | ----------- |
| `model_path` | string | **Required** | The full path to the ONNX file. |
| `label_path` | string | Optional | The full path to the names of the category labels. |

> | `num_threads` | int | Optional | The number of threads. |

Author: ```suggestion
| `num_threads` | int | Optional | The number of threads. Must be 1. |
| `package_reference` | string | Optional | The package of the model to deply. For example: "viam-soleng/face-detector-onnx". |

```

| ------- | ------ | ------------ | ----------- |
| `model_path` | string | **Required** | The full path to the ONNX file. |
| `label_path` | string | Optional | The full path to the names of the category labels. |
| `num_threads` | int | Optional | The number of threads. Must be 1. |

> | `package_reference` | string | Optional | The package of the model to deply. For example: "viam-soleng/face-detector-onnx". |

Reviewer: Do you know what the `package_reference` is? We don't use the field in the config of the module, the module would only know about the `model_path`, which should hold the path to where the model was deployed if the user used the registry to get their model

Author: right that's a problem in the UI as well then. I'll tag you in the ticket

> | `package_reference` | string | Optional | The package of the model to deply. For example: "viam-soleng/face-detector-onnx". |

Author: ```suggestion
| `package_reference` | string | Optional | The package of the model to deploy if the model was deployed through the registry. For example: "viam-soleng/face-detector-onnx". |
```
